docs/Actions.md

```diff
@@ -1063,6 +1073,20 @@ puts release['name']
 
 To get a list of all available values run `fastlane action get_github_release`.
 
+### import_from_git
```

Comment:
Might have to place it somewhere else, as this is in the middle of a `git` section. How about right at the top?